### PR TITLE
[APP-1089] Updates Gem Version To 0.0.5

### PR DESCRIPTION
**Pull request by tlynam**
https://aurorasolar.atlassian.net/browse/APP-1089?atlOrigin=eyJpIjoiOWI3M2NkYzBjM2JiNDQyY2IzNDBiODFjMDE1MGQxOGEiLCJwIjoiaiJ9

### CODE REVIEW

**Review by levlozhkin — CHANGES_REQUESTED** [review]
The code changes are obfuscated by whitespace ones (spaces changed to tabs). Could you remove the whitespace changes?

**Comment by tlynam** [comment]
Sure thing, was battling whitespace issues working in these gems but will try to clean this up so it's easy to read.  Maybe I'll create a couple commits that firsts normalizes it.

**Comment by tlynam** [comment]
Okay @levlozhkin there are two commits.  The first commit is easier to see the changes and the second updates from tabs to spaces and fixes indentation.

File: lib/geometry/arc.rb [hunk 1]
```diff
@@ -13,12 +13,17 @@ module Geometry
...
+class Arc
 	include ClusterFactory
 
 	attr_reader :center
 	attr_reader :radius
 	attr_reader :start_angle, :end_angle
+	attr_writer :options
+	def options
+		@options = {} if !@options
+		@options
```
Comment (levlozhkin) [inline lib/geometry/arc.rb]: Why not:
```ruby
def options
  @options ||= {}
end
```

File: lib/geometry/arc.rb [hunk 2]
```diff
@@ -34,13 +39,13 @@ class Arc
 	# @option options [Numeric] :end	Ending angle
 	# @return [ThreePointArc]
 	def self.new(options={})
```
Comment (levlozhkin) [inline lib/geometry/arc.rb]: Hmm, this is interesting. Having `Arc.new` return a `ThreePointArc` kinda steps on principle of least surprise :thinking:

**Review by levlozhkin — APPROVED** [review]
LGTM, one tiny suggestion. Btw git add interactive mode (`git add -p`) can be helpful in avoiding unexpected whitespace changes.

File: lib/geometry/arc.rb (hunk 1 above)
Comment (tlynam) [reply lib/geometry/arc.rb]: For sure, I had just copied pasted and should have reviewed the code itself for improvements

**Comment by tlynam** [comment]
Thanks @levlozhkin!  I'll be more careful with whitespace going forward so it's more clear and separate it into multiple commits if needed.

**Comment by levlozhkin** [comment]
:+1: Btw, are you pushing with Mitchell's SSH key?:

image

**Pull request merged**